debugtools/DDR_VM/src/com/ibm/j9ddr/CompatibilityConstants29.dat

```diff
@@ -34,6 +34,8 @@
 
 CLimits.USHRT_MAX = 0xFFFF
 
+J9BuildFlags.gc_enableSparseHeapAllocation = 0
```

Comment:
I don't see any DDR code that would need this, but it doesn't work; remove this addition and use `J9VM_GC_ENABLE_SPARSE_HEAP_ALLOCATION` instead (if needed).